crypto/info.c

```diff
@@ -101,10 +102,10 @@ DEFINE_RUN_ONCE_STATIC(init_info_strings)
         add_seeds_string("getrandom-syscall");
 #endif
 #ifdef OPENSSL_RAND_SEED_DEVRANDOM
-        add_seeds_stringlist("random-device", { DEVRANDOM, NULL });
+        add_seeds_stringlist("random-device", DEVRANDOM);
```

Comment:
@levitte I guess that `add_seeds_stringlist()` went into 096978f09908 without ever being expanded, let alone compiled?

Reply:
[ahem] looks that way...